portal/server/controllers/dashboard/index.js

```diff
@@ -30,8 +29,42 @@ const dealsDashboard = async (req, res) => {
     pages,
     successMessage: getFlashSuccessMessage(req),
     primaryNav,
+    tab,
     user: req.session.user,
   });
 };
 
-export default dealsDashboard;
+export const gefDeals = async (req, res) => {
+  const tab = 'gefDeals';
+  const { userToken } = requestParams(req);
+
+  const { count, deals: rawDeals } = await getApiData(
+    api.gefDeals(req.params.page * PAGESIZE, PAGESIZE, {}, userToken),
+    res,
+  );
+
+  const deals = rawDeals.map((deal) => ({
+    _id: deal._id,
+    status: deal.status,
+    bankRef: deal.bankInternalRefName,
+    exporter: deal.exporter.companyName,
+    product: 'GEF',
+    type: '-', // TODO: when types are established this needs to be added into dashboard
```

Comment:
what kind of type do we mean, do you have example? Just thinking of data models

Reply:
type here is AIN, MIN, MIA

Reply:
Ah ok great - i'm adding submissionType to gef deals right now. Portal already has this under `details.submissionType`. Can update this when gef type is in place